tsconfig.json

```diff
@@ -1,12 +1,18 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "allowSyntheticDefaultImports": true,
+    "declaration": true,
+    "esModuleInterop": true,
     "module": "commonjs",
+    "noFallthroughCasesInSwitch": true,
+    "noImplicitReturns": true,
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "outDir": "./dist",
+    "sourceMap": true,
```

Comment:
Source maps are only for Helmet maintainers, right? Is there any benefit to shipping them with npm?

Reply:
Not particularly that I can think of - end users will probably bundle and minify this library with their application anyway, and by that point we're two levels deep into source maps which sounds like a pain. I've added source maps to `.npmignore`.

Reply:
Coming back to this—I'm not sure that end users will bundle/minify a Node library. I could see some tool trying to fill in the source map and failing to do so.

Opened #6 to remove them—feel free to review.